src/lib/components/link.svelte

```diff
@@ -27,6 +26,12 @@
     transform-origin: bottom;
     transition: transform 0.2s ease-in-out;
   }
+  @media only screen and (max-width: 1024px) {
```

Comment:
why do you add this?

Reply:
because for tablet it has a different padding

Reply:
why ?

Reply:
why?

Reply:
because if we put the same padding for all the screens it is very focused and it looks really bad but if you want I put everything the same

Reply:
if you look at the code it isn't actually changing padding, you're changing the display value - this is unexpected behaviour